### ISSUE TITLE
Diffuse lung disease

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#6681]
**Preferred term label:**
Diffuse lung disease

**Synonyms**

**Definition (free text, please give PubMed ID)**
Diseases that involve the entire lungs, i.e. from base to top, peripheral to central and front to back. Can be patchy or uniform.
PMID: 31227433; PMID: 8044098

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 26408013

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Can we use the existing modifier ``Diffuse`` in combination with other terms?

https://hpo.jax.org/app/browse/term/HP:0020034

**Comment by mgriese** [comment]
I don´t think so, as this is a group of "diffuse lung diseases"; e.g. Asthma, which is also "diffuse" according to the modifier would not be classified as a diffuse lung disease.

**Comment by nicolevasilevsky** [comment]
Should this be added as a disease in Mondo, rather than a phenotype?

https://github.com/monarch-initiative/mondo/issues/3026

**Comment by mgriese** [comment]
Yes, as a very high level classifying disease group, in contrast to localized lung diseases (Atelectasis, sequester, etc). DLD includes diffuse parenchymal lung diseases, vascular diseases, (and some airway diseases or some proliferative diseases).

**Comment by pnrobinson** [comment]
@nicolevasilevsky  Let's transfer this to MONDO with your NTR above https://github.com/monarch-initiative/mondo/issues/3026
